packages/paste-core/components/badge/src/index.tsx

```diff
@@ -53,10 +74,24 @@ Badge.displayName = 'Badge';
 Badge.propTypes = {
   children: PropTypes.node.isRequired,
   element: PropTypes.string,
-  variant: PropTypes.oneOf(['info', 'default', 'warning', 'error', 'success', 'new'] as BadgeVariants[]).isRequired,
+  variant: PropTypes.oneOf([
+    'neutral',
+    'warning',
+    'error',
+    'success',
+    'new',
+    'decorative10',
+    'decorative20',
+    'decorative30',
+    'decorative40',
+    'neutral_counter',
+    'error_counter',
+    // the following variants are outdated but still supported to prevent breaking changes
```

Comment:
Suggestion: if these are unsupported, should we take them out of the propTypes?

Reply:
they are still supported, since we want people to still have access to using them if they're already using them. We just don't want anyone else to start using these variants, hence why they're not on the docs. I refer to them as "outdated" to try to separate them from being "deprecated (and therefore no longer supported)"